sys/suit/suit.c

```diff
@@ -0,0 +1,326 @@
+/*
+ * Copyright (C) 2018 Freie Universit�t Berlin
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_suit
+ * @{
+ *
+ * @file
+ * @brief       SUIT manifest parser library
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ *
+ * @}
+ */
+#include <string.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "suit.h"
+#include "cbor.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+static int _advance_x(CborValue *it, unsigned x)
+{
+    for(unsigned i=0; i < x; i++)
+    {
+        cbor_value_advance(it);
+        if (cbor_value_at_end(it)) {
+            return -1;
+        }
+    }
+    return 0;
+}
+
+static int parse_manifest_version(CborValue *it,
```

Comment:
please rename ```parse_*``` to ```_parse_*``` if static.

Reply:
I should know better, changed 5 occurences.